packages/circuit-ui/components/ImageInput/ImageInput.tsx

```diff
@@ -46,15 +46,7 @@ export interface ImageInputProps
    * The visual component to render as an image input. It should accept an src
    * prop to render the image.
    */
-  component: ({
-    src,
-    alt,
-    role,
-  }: {
-    src?: string;
-    alt: string;
-    role: 'presentation';
-  }) => JSX.Element;
+  component: ({ src }: { src?: string }) => JSX.Element;
```

Comment:
```suggestion
  component: (props: { src?: string; 'aria-hidden': "true" }) => JSX.Element;
```

Reply:
Ah, without this, a custom `<Component />` could not render the aria-hidden that we're passing, right? 🙈  let me check of this works

Reply:
4f67974 🙏